.markdownlint.yaml

```diff
@@ -20,13 +20,23 @@ MD010:
 # MD013/line-length : Line length : https://github.com/DavidAnson/markdownlint/blob/v0.32.1/doc/md013.md
 MD013: false
 
+# MD024/no-duplicate-heading Multiple headings with the same content (disabled because we do it).
```

Comment:
we cannot use the same `.markdownlint.yaml` for both the `/docs` and `/`, because the indentation works differently for mkdocs and GitHub. Let's not make this change at this point and optimize linting first for `/docs` and make sure that all bullet points and enumerations are rendered as intended. This is currently not the case.